app/src/usb.c

```diff
@@ -38,6 +38,22 @@ static const struct hid_ops ops = {
     .int_in_ready = in_ready_cb,
 };
 
+#ifdef CONFIG_ZMK_USB_REPORT_LEDS
+
+int zmk_usb_hid_receive_report(uint8_t *report, size_t len) {
```

Comment:
Question for someone more familiar with USB/HID than I am: is it okay to attempt to read regardless of the USB status, or do we need a `switch` like the one in `zmk_usb_hid_send_report()`?